research-plan.md

```diff
@@ -0,0 +1,75 @@
+# FOIA platform research plan
+
+This is a living document. It will be updated over the course of the coming weeks as we learn more from our users and as we hone in our hypotheses.
+
+## Background
+
+Initial research on National FOIA Portal. Attempting to focus on speaking to user groups who were not heavily represented in previous FOIA 1.0 research.
+
+The goal of this research is to inform hypotheses that address the following:
+- How might we create a platform that improves the FOIA requesting experience?
+- How might we create a platform that increases the quality of FOIA requests?
+- How might various FOIA requesters use a national FOIA platform?
+
+## User groups
+
+Prioritization is a function of how much insight I believe we have into a user group.
+
+### Requesters
+
+- Business/Industry (High priority)
+- Attorneys (High priority)
+- Medium priority (Medium priority)
+- Civic society (Low priority)
+- Journalists (Low priority)
+
+### Agencies
+
+- High volume
+- Low volume
+- Home grown FOIA system
+- Agencies using major FOIA platforms
+- High security/high-side systems
+
+### Software vendors
+
+## Methods
+
+- 30-45m interviews
+- Journey map: Mapping how of agencies and requesters think the process works
+- Service blueprint: Neutral map documenting current process from both agency and requester perspectives.
+- Usability testing (once prototype is built)
+- Contextual inquiry (if time allows): Observing a FOIA requester as they file a request, follow up on existing request, and receive their responsive documents
+- Diary study (if time allows): Asking a set of users to log all FOIA related activities over the lifecycle of a request.
+
+## Interview guide
+
+Note: Avoid detailing project scope until after the interview is complete.
+
+### Requesters
+
+- How often do you file FOIA requests?
+- How do you decide to file a FOIA request?
+- What are some things you do before filing a FOIA request?
+- (If not addressed above: Do you search for existent publicly released documents before filing a FOIA request?)
+- How do you identify which agency to submit your request
+- How do you identify where to submit a request to an agency?
+- How do you identify how to submit a request to an agency?
+- Have you ever sent the same request to multiple agencies?
+- How do you track the progress of your request?
+- Have you experienced stalled requests? If so, what have you done in response?
+- What agencies have you interacted with that you have had a particular good experience with regard to FOIA?
+- Have you ever had a particularly frustrating experience with a FOIA request? (Tell us about it)
+- If you could wave a magic wand to address one issue in the FOIA space, what would that issue be?
+
+### Agency
+
+- What percentage of your requests are:
```

Comment:
I still think there's value in asking this. We can stop if we feel it's redundant.